src/features/dfdElements/outputPortBehaviorValidation.ts

```diff
@@ -23,13 +23,13 @@ export class PortBehaviorValidator {
     // Regex that validates a set statement.
     // Has the label type and label value that should be set as capturing groups.
     private static readonly SET_REGEX =
-        /^set +([A-Za-z][A-Za-z0-9_]*)\.([A-Za-z][A-Za-z0-9_]*) *= *(?: +|!|TRUE|FALSE|\|\||&&|\(|\)|[A-Za-z][A-Za-z0-9_]*(?:\.[A-Za-z][A-Za-z0-9_]*){2})+$/;
+        /^set +([A-Za-z][A-Za-z0-9_]*)\.([A-Za-z][A-Za-z0-9_]*) *= *(?: +|!|TRUE|FALSE|\|\||&&|\(|\)|[A-Za-z][A-Za-z0-9_\|]*(?:\.[A-Za-z][A-Za-z0-9_\|]*){2})+$/;
```

Comment:
The regex matches `set sensitivity.private = auth|server.sensi|tivity.priv|ate`
The last to pipes have to be remove as we don't want to allow pipes in labels